src/xb-silo-export.c

```diff
@@ -38,22 +38,22 @@ xb_silo_export_node (XbSilo *self, XbSiloExportHelper *helper, XbSiloNode *sn, G
 				xb_silo_from_strtab (self, sn->element_name));
 
 	/* add any attributes */
-	for (guint8 i = 0; i < sn->nr_attrs; i++) {
-		XbSiloNodeAttr *a = xb_silo_get_attr (self, helper->off, i);
+	for (guint8 i = 0; i < xb_silo_node_get_nr_attrs (sn); i++) {
```

Comment:
Nitpick: `xb_silo_node_get_n_attrs()` or `xb_silo_node_get_attr_count()` would be more conventional ways of naming this. There are no examples of `*_get_nr_*` function names in devhelp for me.

Reply:
xb_silo_node_get_attr_count works much better, thanks!